lawlibrary/views/legislation.py

```diff
@@ -55,5 +57,6 @@ def get_context_data(self, **kwargs):
             self.navbar_link = "legislation/municipal"
             context["breadcrumb_link"] = reverse("municipal_legislation")
             context["legislation_list_show_dates"] = True
+            context["entity_profile"] = False
```

Comment:
This isn't necessary I don't think -- the base class already handles entity profiles for localities.